gradebookng/tool/src/webapp/styles/gradebook-grades.css

```diff
@@ -418,15 +418,17 @@ div.wicket-modal div.w_right > div {
 .gb-summary-grade-panel .gb-summary-category-toggle:hover,
 .gb-summary-grade-panel .gb-summary-category-toggle:active {
   text-decoration: none;
-  float: left;
+  display: flex;
```

Comment:
Flexbox not fully supported in IE browsers yet. I can't remember if we have a set of supported browsers?

Reply:
We try our best to support IE 11+ and Edge.  I see plenty of use of display: flex in library/

Reply:
Flex should be fine in Edge. I tested this in IE11 and I didn't see any issues.

Reply:
Flex works great in all Sakai supported browsers